README.md

```diff
@@ -124,6 +133,11 @@ match result {
 
 We want to give credit to [gaffer on games](https://gafferongames.com/) as we have used his guide to building a game networking protocol to build this library. 
 
+## Note on Stability
+
+This library is not fully stable yet. 
+Although version 0.1.0 is released we might have to change some of the existing API.  
```

Comment:
I would mention something about it being used in amethyst as an example usage too.